xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -1488,19 +1488,170 @@ bool CGUIMediaWindow::OnPlayMedia(int iItem, const std::string &player)
  *
  * This function is called by OnClick()
  */
+class QueueMediaAsync : public CJob
+{
+public:
+  QueueMediaAsync(int iPlaylist, std::unique_ptr<CFileItemList> Localm_vecItems)
+  {
+    m_iPlaylist = iPlaylist;
+    m_Localm_vecItems.reset(new CFileItemList);
+    m_Localm_vecItems->Copy(*Localm_vecItems);
+  }
+
+  ~QueueMediaAsync(void) {}
+
+  bool DoWork(void)
+  {
+    bool ItemDetected = false;
+    int PlaylistSize = 0;
+    int Localm_vecItemsSize = m_Localm_vecItems->Size();
+
+    for ( int i = 0; i < Localm_vecItemsSize; i++ )
```

Comment:
No blanks after opening bracket and before closing bracket. Here and at other places.